contents/docs/migrate/_snippets/migration-warning.mdx

```diff
@@ -0,0 +1,6 @@
+> Prior to starting a historical data migration, ensure you do the following:
+> 1. Create a project on our [US](https://us.posthog.com/signup) or [EU](https://eu.posthog.com/signup) Cloud
+> 2. Sign up to a paid product analytics plan on [the billing page](https://us.posthog.com/organization/billing) (historic imports are free but this unlocks the necessary features).
+> 3. Raise an [in-app support request](http://us.posthog.com/home#supportModal) (Target Area: Data Management) detailing where you are sending events from and the total volume.
```

Comment:
where = self-hosted posthog or amplitude or mixpanel ...
how = using our migration scripts, using a custom python script, using airflow, ...
speed = 10k/minute
maybe there's a better way to describe this? Giving people an example template?
```suggestion
> 3. Raise an [in-app support request](http://us.posthog.com/home#supportModal) (Target Area: Data Management) detailing where you are sending events from, how and the total volume and speed.
```

Reply:
updated with an example